test/unit/tracks/text-track-controls.test.js

```diff
@@ -435,7 +436,11 @@ test('chapters should be displayed when remote track added and load event fired'
 
   equal(chaptersEl.track.cues.length, 2);
 
-  TestHelpers.triggerDomEvent(chaptersEl, 'load');
+  if (player.tech_.featuresNativeTextTracks) {
+    TestHelpers.triggerDomEvent(chaptersEl, 'load');
```

Comment:
On IE8 we use a custom DOM element for text tracks, however, IE8 doesn't allow us to trigger custom events on a custom element.
So, because everywhere we support native text tracks, we can trigger a real DOM event, we use that and where we use emulated tracks, we just trigger `load` directly on the chapters element object.